var/spack/repos/builtin/packages/ultraliser/package.py

```diff
@@ -9,14 +9,11 @@
 class Ultraliser(CMakePackage):
     """Mesh and volume reconstruction of neuroscientific models
     """
-    homepage = "https://bbpcode.epfl.ch/code/#/admin/projects/viz/Ultraliser"
-    git = "ssh://bbpcode.epfl.ch/viz/Ultraliser"
+    homepage = "https://github.com/BlueBrain/Ultraliser"
+    git = "git@github.com:BlueBrain/Ultraliser.git"
 
-    version('0.2.0', tag='v0.2.0')
+    version('0.3.0', tag='v0.3.0')
```

Comment:
CI is failing because no such tag available on GitHub : https://github.com/BlueBrain/Ultraliser/tags

Reply:
Ok. I will tag it.